code/datums/brain_damage/mild.dm

```diff
@@ -0,0 +1,278 @@
+//Mild traumas are the most common; they are generally minor annoyances.
+//They can be cured with mannitol and patience, although brain surgery still works.
+//Most of the old brain damage effects have been transferred to the dumbness trauma.
+
+/datum/brain_trauma/mild
+
+/datum/brain_trauma/mild/hallucinations
+	name = "Hallucinations"
+	desc = "Patient suffers constant hallucinations."
+	scan_desc = "schizophrenia"
+	gain_text = "<span class='warning'>You feel your grip on reality slipping...</span>"
+	lose_text = "<span class='notice'>You feel more grounded.</span>"
+
+/datum/brain_trauma/mild/hallucinations/on_life()
+	owner.hallucination = min(owner.hallucination + 10, 50)
+	..()
+
+/datum/brain_trauma/mild/hallucinations/on_lose()
+	owner.hallucination = 0
+	..()
+
+/datum/brain_trauma/mild/stuttering
+	name = "Stuttering"
+	desc = "Patient can't speak properly."
+	scan_desc = "reduced mouth coordination"
+	gain_text = "<span class='warning'>Speaking clearly is getting harder.</span>"
+	lose_text = "<span class='notice'>You feel in control of your speech.</span>"
+
+/datum/brain_trauma/mild/stuttering/on_life()
+	owner.stuttering = min(owner.stuttering + 5, 25)
+	..()
+
+/datum/brain_trauma/mild/stuttering/on_lose()
+	owner.stuttering = 0
+	..()
+
+#define BRAIN_DAMAGE_FILE "brain_damage_lines.json"
```

Comment:
This is already defined elsewhere

Reply:
fixed

Reply:
The value is used elsewhere, just put the define in one of the defines files

Reply:
Why is it even a define? All it saves is writing a .json at the end, while clouding which file it actually is to code readers

Reply:
because the exact location and name of the file can more easily be changed from one location then